compiler/ballerina-lang/src/main/java/io/ballerina/projects/Module.java

```diff
@@ -87,6 +87,10 @@ public ModuleDescriptor descriptor() {
         return new DocumentIterable(this.testSrcDocs.values());
     }
 
+    public byte[] birBytes() {
+        return this.moduleContext.getBirBytes();
```

Comment:
I don't see any other compiler outputs (such as getBLangPackageOrThrow) exposed from this class, Isn't it enough to expose this form the `ModuleContext`

Reply:
I have removed exposing bir from Module. This was only used for test bir. Added api in nullbackend (which is use for test bir purpose) to get default module bir.